storage/rocksdb/rdb_mutex_wrapper.h

```diff
@@ -75,7 +75,11 @@ class Rdb_cond_var : public rocksdb::TransactionDBCondVar {
   Rdb_cond_var &operator=(const Rdb_cond_var &) = delete;
 
 public:
+#ifdef HAVE_PSI_INTERFACE
+  Rdb_cond_var(PSI_memory_key psi_key);
```

Comment:
Another nice catch, yes, this should definitely be explicit. I'll make these changes then merge.